Modules/SaltStackConfig/SaltStackConfig.Format.ps1xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+    <ViewDefinitions>
+        <View>
+            <Name>SscConnection</Name>
+            <ViewSelectedBy>
+                <TypeName>SscConnection</TypeName>
+            </ViewSelectedBy>
+            <TableControl>
+                <TableHeaders>
+                    <TableColumnHeader>
+                        <Width>25</Width>
+                        <Label>Server Name</Label>
```

Comment:
I would recommend to match the column header with the name of the property. When the user sees the column header (for example "ServerName") they would normally expect to be able to refer to it as a property of the object ($server.ServerName). For the same reason I would recommend that you make the other two first level properties of the server object.

Reply:
Understood, I've made this change in the most recent commit.